@navikt/core/react/src/alert/Alert.tsx

```diff
@@ -71,27 +84,45 @@ export const Alert = forwardRef<HTMLDivElement, AlertProps>(
       size = "medium",
       fullWidth = false,
       inline = false,
+      closeButton = false,
+      onClose,
       ...rest
     },
     ref
-  ) => (
-    <div
-      {...rest}
-      ref={ref}
-      className={cl(
-        className,
-        "navds-alert",
-        `navds-alert--${variant}`,
-        `navds-alert--${size}`,
-        { "navds-alert--full-width": fullWidth, "navds-alert--inline": inline }
-      )}
-    >
-      <Icon variant={variant} className="navds-alert__icon" />
-      <BodyLong as="div" size={size} className="navds-alert__wrapper">
-        {children}
-      </BodyLong>
-    </div>
-  )
+  ) => {
+    return (
+      <div
+        {...rest}
+        ref={ref}
+        className={cl(
+          className,
+          "navds-alert",
+          `navds-alert--${variant}`,
+          `navds-alert--${size}`,
+          {
+            "navds-alert--full-width": fullWidth,
+            "navds-alert--inline": inline,
+          }
+        )}
+      >
+        <Icon variant={variant} className="navds-alert__icon" />
+        <BodyLong as="div" size={size} className="navds-alert__wrapper">
+          {children}
+        </BodyLong>
+        {closeButton && !inline && (
+          <div className="navds-alert__button-wrapper">
+            <Button
```

Comment:
Ser ut som en bug henger igjen fra endringen til `focus-visible` der man får 1 px white border ved active, selv om man ikke har fokus. Usikker på om dette gjelder alle knappene eller bare tertiary-neutral, men vært nice å få fikset i samme slengen
<img width="131" alt="Screenshot 2023-06-26 at 15 32 38" src="https://github.com/navikt/aksel/assets/26967723/68d7eabe-4296-4f04-ab14-eb5d0a3ce952">

Reply:
Hvordan er desired look for knapper under activation? (`:active`)? Ser ikke noe slikt under Button i figma (core UI kit) 🤔. Det ser ut som at vi har en inset white border på knapper der de har en solid bakgrunnsfarge, men er det slik at den skal _ikke_ legges på der en knap _endrer_ fargen på bakgrunnen? (feks i tertiary sin `:active` state)

Reply:
Inset 1px white var lagt til for å skape et visuelt skille mellom fokusmarkering og knapp ved siden man alltid fikk `:focus` tidligere når `:active` ble aktivert. Nå er det et bedre skille der `:active`, `:focus-visible` og `:active:focus-visible` er 3 separate states.
Så inset trengs bare nå ved  `:active:focus-visible`